indexing-service/src/main/java/io/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -186,13 +198,28 @@ public Response getWorkerConfig()
   @Path("/worker")
   @Consumes(MediaType.APPLICATION_JSON)
   public Response setWorkerConfig(
-      final WorkerBehaviorConfig workerBehaviorConfig
+      final WorkerBehaviorConfig workerBehaviorConfig,
+      @QueryParam("author") String author,
```

Comment:
can we have these fields as part of a new serializable object? This way we can add to this object